ssr/style/input.css

```diff
@@ -16,4 +16,20 @@
         --color-primary-900: 115 2 62;
         --color-primary-950: 80 1 43;
     }
+}
+
+@media (min-height:820px) and (max-height:899px){
+    .hot-left-arrow{
+        margin-top: 206px !important;
+    }
+}
+@media (min-height:768px) and (max-height:819px){
+    .hot-left-arrow{
+        margin-top: 166px !important;
+    }
+}
+@media (max-height:767px){
+    .hot-left-arrow{
+        margin-top: 75px !important;
+    }
```

Comment:
Does this not work?
```
mt-20 md:mt-40 lg:mt-52 
```
any specific reason why we need these specific values for screen heights `820 to 899`?

Reply:
I think sm, md.. breakpoints are on screen widths only.  Used these breakpoints in the screen heights for handing arrow's position.